.circleci/config.yml

```diff
@@ -0,0 +1,38 @@
+# Javascript Node CircleCI 2.0 configuration file
```

Comment:
You might also want to checkout your `package.json` and `.gitignore` for your `add-linting` PR.